GetSubscriptionsFeed/handler.ts

```diff
@@ -127,14 +135,20 @@ export function GetSubscriptionsFeedHandler(
 
     const subscriptions = new Array<FiscalCodeHash>();
     profileSubscriptionsSet.forEach(ps => {
-      if (!serviceUnsubscriptionsSet.has(ps)) {
+      if (
+        !serviceUnsubscriptionsSet.has(ps) &&
+        !profileUnsubscriptionsSet.has(ps)
```

Comment:
I think this check is useless since you either have a profile subscription OR a profile unsubscription but never both in the same day

Reply:
It is true if we mantain 7 delay days between request and effectiveness account's deletion.

Reply:
I think it's true anyway since when you create the (un)subscription record in the same day, you delete the any previous (un)subscription here: https://github.com/pagopa/io-functions-app/blob/master/UpdateSubscriptionsFeedActivity/index.ts#L102

Reply:
Deal

Reply:
Fixed in [8857930](https://github.com/pagopa/io-functions-services/pull/86/commits/885793016f5a3b97cae33ed45819ab596d870f67)